testing/openttd/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Adrian Siekierka <kontakt@asie.pl>
+# Maintainer: Adrian Siekierka <kontakt@asie.pl>
+pkgname=openttd
+pkgver=1.6.1
+pkgrel=0
+pkgdesc="Open source simulation game based upon Transport Tycoon Deluxe"
+url="http://openttd.org/"
+arch="all"
+license="GPL2"
+depends=""
```

Comment:
Actually, this is correct - OpenGFX and OpenSFX are not strictly required for OpenTTD to run and, if they're not installed globally, OpenTTD will ask if you want to download and install them locally. You can also use the original assets from Transport Tycoon Deluxe, or a different project called zBase.